src/stan/math/prim/scal/meta/is_vector_like.hpp

```diff
@@ -10,7 +10,7 @@ namespace stan {
   // handles scalar, eigen vec, eigen row vec, std vec
   template <typename T>
   struct is_vector_like {
-    enum { value = stan::is_vector<T>::value };  
+    enum { value = false };  
```

Comment:
why is this being changed? Maybe this is why you needed to define `is_vector_like` for `std::vector`? Does this not work for `MatrixBase` by just adding a template specialization for `MatrixBase`?